apps/perf-test-react-components/.eslintrc.json

```diff
@@ -4,6 +4,6 @@
   "rules": {
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=32960&commitId=06f043fe06b4da25ff9b475aa5b5ac47e0c27add&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.No callout width specified.chromium.png|2319| Changed|
|Callout.Rendering callout attached to a rectangle.chromium.png|1493| Changed|
|Callout.Gap space 25.chromium.png|2309| Changed|
|Callout.Bottom Left Edge RTL.chromium.png|2178| Changed|

</details>


  <details><summary><b>react-charting-HeatMapChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-HeatMapChart.Basic Dark Mode.chromium.png|334| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>